refinery/core/serializers.py

```diff
@@ -121,12 +121,23 @@ def partial_update(self, instance, validated_data):
 class ExtendedGroupSerializer(serializers.ModelSerializer):
     member_list = serializers.SerializerMethodField()
     perm_list = serializers.SerializerMethodField()
+    can_edit = serializers.SerializerMethodField()
     uuid = serializers.SerializerMethodField()
     name = serializers.CharField(
         min_length=3,
         validators=[UniqueValidator(queryset=ExtendedGroup.objects.all())]
     )
 
+    def get_can_edit(self, group):
+        user = self.context.get('user')
+        if user is None:
+            return ''
```

Comment:
The name `get_can_edit()` suggests a boolean return value but it can also return an empty string?